packages/ozone/src/mod-service/index.ts

```diff
@@ -757,6 +784,14 @@ export class ModerationService {
       )
     }
 
+    if (onlyMuted) {
+      builder = builder.where((qb) =>
+        qb
+          .where('muteUntil', '>', new Date().toISOString())
+          .orWhere('muteReportingUntil', '>', new Date().toISOString()),
+      )
+    }
```

Comment:
I expected to see `muteReportingUntil` in here, but I don't totally understand what `muteUntil` is doing in here too.  What's the right way to think about it?

Reply:
In my mind, `Mute` is just one state applied in 2 different fronts. Either the subject is muted from showing up in queue regardless of state change or the subject is muted from changing state of other subjects through reports. With that explained, `onlyMuted` filter is meant to show *all subjects* that are `muted` which is why both `muteUntil` and `muteReportingUntil` are checked here.